src/lib/global.ts

```diff
@@ -5,29 +5,49 @@ import { Client, ClientType } from "src/lib/client"
 import { AutoLauncher } from "src/lib/native"
 
 
-let text: any
+let text: { [index: string]: { [index: string]: string } } = {}
 let client!: ClientType
 let launchOnBoot!: AutoLauncher
-export var data = reactive({ status: { state: "loading", message: "loading" }, loc: { selected: 'en', text, }, client, launchOnBoot })
-export var mutations = {
-  async initClient() {
-    data.client = null
+
+
+export interface GlobalMutations {
+  initClient(): void
+  initLaunchOnBoot(): void
+  changeLang(newLang: string): void
+  loadLangData(): void
+}
+export class Global {
+  data = reactive({ status: { state: "loading", message: "loading" }, loc: { selected: 'en', text, }, client, launchOnBoot })
+  mutations: GlobalMutations = {
+    initClient: this.initClient,
+    initLaunchOnBoot: this.initLaunchOnBoot,
+    changeLang: this.changeLang,
+    loadLangData: this.loadLangData
+  }
+  protected async initClient() {
+    this.data.client = null
     const client = await Client()
```

Comment:
This is somewhat confusing too. I'd expect either `new Client()` or `await Client.init()`, but not `await Client()`

Reply:
It's because Client is not a class. I can convert it to a class but I don't want to modify it until after we have merged in the changes from Leo.

Reply:
I agree, let's have it as is for now